stdlib/hashmap.mc

```diff
@@ -25,17 +29,13 @@ let _hashmapDefaultBucketCount = 100
 let _hashmapBucketIdx = lam hash. lam hm. modi (absi hash) (length hm.buckets)
 
 
--- Returns an empty hashmap with a default number of buckets.
---   eq: A function that specifies equalities between keys in the hashmap.
---   hashfn: A function for computing the hash of a key value.
-let hashmapEmpty = lam eq. lam hashfn.
+-- 'hashmapEmpty ()' returns an empty hashmap with a default number of buckets.
+let hashmapEmpty : Unit -> HashMap = lam _.
```

Comment:
I do not understand this change. Why does this have to be evaluated at runtime?

Reply:
You are right, I fixed this.